types/node/crypto.d.ts

```diff
@@ -46,8 +46,21 @@ declare module "crypto" {
         digest(): Buffer;
         digest(encoding: HexBase64Latin1Encoding): string;
     }
+
+    interface KeyObject {
+        asymmetricKeyType?: KeyType;
+        export(options?: {
+            type: 'pkcs1' | 'spki' | 'pkcs8' | 'sec1',
+            format: KeyFormat,
+            cipher?: string,
+            passphrase?: string
+        }): string | Buffer;
+        symmetricSize: number;
```

Comment:
```suggestion
        symmetricSize?: number;
```
see https://nodejs.org/docs/latest-v11.x/api/crypto.html#crypto_keyobject_symmetrickeysize